From 886dfd99c707b42cc6e20dc75cec8d52c383f964 Mon Sep 17 00:00:00 2001
From: Nathan Menhorn <nathan.menhorn@amd.com>
Date: Tue, 16 Jan 2024 15:59:17 -0800
Subject: [PATCH 2/2] Corrected non-secure Payload Address

When fixing the opteed_fast error, the non-secure memory address also
needs to be updated as well.

Signed-off-by: Nathan Menhorn <nathan.menhorn@amd.com>
---
 plat/xilinx/versal_net/bl31_versal_net_setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/xilinx/versal_net/bl31_versal_net_setup.c b/plat/xilinx/versal_net/bl31_versal_net_setup.c
index f606fd68b..c4e6e96ed 100644
--- a/plat/xilinx/versal_net/bl31_versal_net_setup.c
+++ b/plat/xilinx/versal_net/bl31_versal_net_setup.c
@@ -55,7 +55,7 @@ static inline void bl31_set_default_config(void)
 {
 	bl32_image_ep_info.pc = 0x22200000uL;//BL32_BASE;
 	bl32_image_ep_info.spsr = arm_get_spsr_for_bl32_entry();
-	bl33_image_ep_info.pc = plat_get_ns_image_entrypoint();
+	bl33_image_ep_info.pc = 0x26300000uL;//plat_get_ns_image_entrypoint();
 	bl33_image_ep_info.spsr = SPSR_64(MODE_EL2, MODE_SP_ELX,
 					DISABLE_ALL_EXCEPTIONS);
 }
-- 
2.34.1

